Portfolio/index.html

```diff
@@ -7,7 +7,34 @@
   </head>
```

Comment:
There is no need to surround the 'Deo' text with <h3> tags. Just the text itself is sufficient.